labs/project-juno-moneta.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Project Juno Moneta
+parent: Labs
+---
+# Lab Name
+Project Juno Moneta
+
+# Short Description
+Project Juno Moneta is an experimental blockchain based open-source CBDC platform for different wholesale and crossborder usecases. 
+
+# Scope of Lab
+
+The scope of this lab is to provide different experimental wholesale and crossborder CBDC applications on the top of Hyperledger Besu. The project investigates and implements on a prototype basis many CBDC usescases, like
+
+- crossborder wholesale CBDC
+- digitized commercial bank money
+- general purpose governance module
+- payment versus payment
+- payment versus delivery
+- tokenized foreign exchange 
+- green finance 
+- ...
+
+# Initial Committers
+
+https://github.com/Daniel-Szego
+https://github.com/hbalazs1980
+
+# Sponsor
+
+
+# Pre-existing repository
+
+https://github.com/Daniel-Szego/Project-Juno-Moneta
```

Comment:
It looks like some of the commits do not have proper sign-offs. Here is a guide on [how to sign previous commits](https://hyperledger-indy.readthedocs.io/projects/sdk/en/latest/docs/contributors/signing-commits.html#how-to-sign-previous-commits). Happy with this lab once that is done.

Reply:
Thanks for your feedback and effort, I have recreated the preexisting reporsitory, having one branch, one-commit, with a signoff from me.